feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-normalizer.spec.ts

```diff
@@ -1148,4 +1148,52 @@ describe('OccConfiguratorVariantNormalizer', () => {
       expect(values[0].selected).toBe(false);
     });
   });
+
+  describe('getPlaceholder', () => {
+    const domainValuesWithIntervals: OccConfigurator.Value[] = [
+      { key: valueKey, name: '2-5' },
+      { key: valueKey, name: '10-12' },
+    ];
+
+    const occAttributeWithIntervalValues: OccConfigurator.Attribute = {
+      name: 'attributewithIntervals',
+      required: requiredFlag,
+      type: OccConfigurator.UiType.NUMERIC,
+      key: groupKey,
+      intervalInDomain: true,
+      domainValues: domainValuesWithIntervals,
+    };
+
+    const configuration: OccConfigurator.Configuration = {
+      configId: configId,
+      complete: true,
+      consistent: true,
+      rootProduct: 'CONF_PRODUCT',
+      groups: [
+        {
+          attributes: [occAttributeWithIntervalValues],
+          groupType: OccConfigurator.GroupType.CSTIC_GROUP,
+          id: '1',
+        },
+      ],
+    };
+
+    it('should not fill placeholder if isIntervalInDomain is false', () => {
+      const result = occConfiguratorVariantNormalizer.getPlaceholder(
+        domainValuesWithIntervals,
+        false
+      );
+      expect(result).toBeUndefined;
```

Comment:
You probably meant to have
```suggestion
      expect(result).toBeUndefined();
```
Without brackets the check is executed at all. Please correct and then fix the test